pkgs/tools/security/swtpm/default.nix

```diff
@@ -100,5 +100,6 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/stefanberger/swtpm";
     license = licenses.bsd3;
     maintainers = [ maintainers.baloo ];
+    mainProgram = finalAttrs.pname;
```

Comment:
This is supposed to be static, much like the `repo` attribute in the `fetchFromGitHub` function. See #275550.

Reply:
Makes sense, changed to be static